lapce-data/src/document.rs

```diff
@@ -773,14 +776,36 @@ impl Document {
                                     style.style,
                                 );
                             }
-                            let styles_span = Arc::new(styles_span.build());
+
+                            let styles = styles_span.build();
+
+                            let mut syntactic_styles_spans = SpansBuilder::new(len);
+
+                            if let Some(syntactic_styles) = syntactic_styles {
+                                for (interval, style) in syntactic_styles.iter() {
+                                    syntactic_styles_spans
```

Comment:
Any reason you want to rebuild the styles spans here?

Reply:
thanks, didn't realise that. 

No reason... just an oversight